easier68k/core/opcodes/ori.py

```diff
@@ -103,11 +103,9 @@ def execute(self, simulator: M68K):
             msb_bit = 0x8000
         elif self.size is OpSize.LONG:
             msb_bit = 0x80000000
-
-        simulator.set_condition_status_code(ConditionStatusCode.N, msb_bit & result_unsigned != 0)
-        simulator.set_condition_status_code(ConditionStatusCode.Z, result_unsigned == 0)
-        simulator.set_condition_status_code(ConditionStatusCode.V, False)
-        simulator.set_condition_status_code(ConditionStatusCode.C, False)
+        
+        # set hte FONCKin CCR
+        simulator.set_ccr_reg('i', (msb_bit & result_unsigned != 0), (result_unsigned == 0), 0, 0)
```

Comment:
I'm guessing this why the `isinstance` assertions were failing, since you were passing in `0`s. We would probably be fine without them, but personally I'd prefer using booleans.

Reply:
Yes that was the problem.
At the moment I am casting to Boolean, which should have similar effects in filtering out invalid input.
To be clear, you are requesting that T/F should be used instead of passing 1 or 0 to the function?

Reply:
Hey Chris, 
When you get some spare time please let me know if the current version works for you :>

Reply:
Sorry for the delay! Unless I can get to it later tonight, I'm most likely going to be unavailable until Wednesday.